assets/base.css

```
@@ -1509,21 +1542,43 @@ details[open] > .share-button__fallback {
   -webkit-appearance: none;
   appearance: none;
   background-color: rgb(var(--color-background));
-  border: 0.1rem solid transparent;
-  border-radius: var(--inputs-radius);
   color: rgb(var(--color-foreground));
   font-size: 1.6rem;
   width: 100%;
-  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground), var(--inputs-border-opacity));
-  height: 4.5rem;
   box-sizing: border-box;
   transition: box-shadow var(--duration-short) ease;
-  filter: drop-shadow(
-    var(--inputs-shadow-horizontal-offset)
-    var(--inputs-shadow-vertical-offset)
-    var(--inputs-shadow-blur-radius)
-    rgba(var(--color-shadow), var(--inputs-shadow-opacity))
-  );
+  border-radius: calc((var(--inputs-border-width)) + (var(--inputs-radius)));
+  min-height: calc(4rem + (var(--inputs-border-width)) * 2);
+  min-width: calc(7rem + (var(--inputs-border-width)) * 2);
```

Parens in the wrong place but I think we can exclude them anyway.

```suggestion
  min-height: calc(4rem + (var(--inputs-border-width) * 2));
  min-width: calc(7rem + (var(--inputs-border-width) * 2));
```